pkg/daemon/daemon.go

```diff
@@ -177,6 +177,42 @@ func (d *Daemon) WaitUntilRunning() error {
 	)
 }
 
+func (d *Daemon) WaitUntilInit() error {
```

Comment:
Better to unify to `WaitUntil(DaemonStateInit)` `WaitUntil(DaemonStateReady)`?

Reply:
Good idea

Reply:
fixed